Doc/library/shlex.rst

```diff
@@ -102,6 +102,11 @@ The :mod:`shlex` module defines the following functions:
       >>> command
       ['ls', '-l', 'somefile; rm -rf ~']
 
+   .. note::
+
+      The :func:`quote` function will not quote shell keywords.
+      (like echo, done, while, etc.)
```

Comment:
I think that saying it doesn't quote shell keywords makes one wonder why shell keywords specifically. It might be better to say something like this instead ([source](https://github.com/python/cpython/blob/f58631be1148ee5436bb71d175d3993e3e6b4e1e/Lib/shlex.py#L321)), but I'll defer to others if this is too much detail:

```suggestion
      The :func:`quote` function will not quote any ASCII word characters, which
      include shell keywords like ``echo``, ``done``, ``while``, etc.
```